src/net_manager.cpp

```diff
@@ -73,6 +74,14 @@ void NetManagerTask::begin()
   }
 }
 
+void NetManagerTask::end()
+{
+  if(NULL != _instance)
+  {
+    MicroTask.stopTask(_instance);
```

Comment:
You will need to explicitly shutdown the networking (after stopping the task), there is no equivalent of setup in MicroTasks

Reply:
what would be the method to shutdown ETH ?